src/static-build/index.tsx

```diff
@@ -60,6 +63,27 @@ const toOutput: Output = {
         purpose: 'maskable',
       },
     ],
+    description:
+      'Compress and compare images with different codecs, right in your browser.',
+    lang: 'en',
+    categories: ['photo', 'productivity', 'utilities'],
```

Comment:
@beaufortfrancois are these from a particular list of categories?

Reply:
Indeed. it comes from https://github.com/w3c/manifest/wiki/Categories#standardized-categories

Reply:
https://www.w3.org/TR/manifest-app-info/#categories-member